packages/storybook/stories/va-segmented-progress-bar.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { getWebComponentDocs, propStructure } from './wc-helpers';
+
+const segmentedProgressBarDocs = getWebComponentDocs('va-segmented-progress-bar');
+
+export default {
+  title: 'Components/va-segmented-progress-bar',
+};
+
+const Template = ({ 'enable-analytics': enableAnalytics, current, total, 'aria-label': ariaLabel }) => (
```

Comment:
Not sure where you would want to put it, but could we add a top and bottom margin in here somewhere similar to what we have in the [stories.jsx](https://github.com/department-of-veterans-affairs/component-library/blob/7ac1818bfe1800703feb2238ce8e78c732e34542/packages/storybook/stories/SegmentedProgressBar.stories.jsx#L10) file for the React segmented progress bar?

When I pop the story for this component into its own window it's very hard to see:
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/12739849/144123920-b01cc1a0-f389-4fd4-a6f9-10f86de41c07.png">

Reply:
Updated to give it breathing room in Storybook and added a comment about why we are doing this here

Reply:
Looks great, thank you!